02_parallel.pl

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env perl
+
+###########################################################################
+#                                                                         #
+# Program     02_parallel.pl                                              #
+# Author      Oleksandr Osavchuk                                          #
+# Description                                                             #
+#             Run user-selected command on many servers (user provided as #
+#             param) with ssh in parallel, collect output from all nodes. #
+#             Script should print collected output from all nodes on      #
+#             stdout, w/o using temp files.                               #
+#                                                                         #
+#                                                                         #
+###########################################################################
+
+use strict;
+use warnings;
+use POSIX qw/strftime/;
+use Getopt::Std;
+use File::Basename;
+use English qw( -no_match_vars );
+use Parallel;
+
+my $USER_COMMAND = '/bin/ps';
```

Comment:
The `$USER_COMMAND` value needs to be passed as an argument to the program. Just because it's "user command"